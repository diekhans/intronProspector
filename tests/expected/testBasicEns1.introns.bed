10	4966081	5097433	sj0	1	.
10	4968944	5102100	sj1	1	.
10	5000666	5001513	sj2	1	.
10	5000666	5132601	sj3	1	.
10	5096577	5097433	sj4	1	.
10	5097550	5098801	sj5	2	.
10	5098879	5099326	sj6	1	.
10	5099449	5102100	sj7	1	.
10	5102650	5105594	sj8	2	.
10	5312165	5454947	sj9	2	.
10	5453560	5454264	sj10	41	.
10	5454522	5454947	sj11	55	.
10	5455118	5456086	sj12	45	.
10	5456273	5456587	sj13	46	.
