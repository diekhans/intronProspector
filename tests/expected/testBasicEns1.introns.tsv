chrom	intron_start	intron_end	strand	uniq_mapped_count	multi_mapped_count	unsure_mapped_count	max_left_overhang	max_right_overhang
10	4966081	5097433	.	0	1	0	72	29
10	4968944	5102100	.	0	1	0	15	85
10	5000666	5001513	.	0	1	0	81	20
10	5000666	5132601	.	0	1	0	81	20
10	5096577	5097433	.	0	1	0	72	29
10	5097550	5098801	.	0	2	0	85	16
10	5098879	5099326	.	1	0	0	80	17
10	5099449	5102100	.	0	1	0	15	85
10	5102650	5105594	.	0	2	0	75	26
10	5312165	5454947	.	0	2	0	10	91
10	5453560	5454264	.	41	0	0	68	86
10	5454522	5454947	.	53	2	0	87	91
10	5455118	5456086	.	45	0	0	85	90
10	5456273	5456587	.	46	0	0	86	79
